docs/cases/indicators-of-compromise.asciidoc

```diff
@@ -130,4 +130,41 @@ When you attach an indicator to a case, the indicator is added as a new comment
 To remove an indicator attached to a case, click *More actions* (*...​*) -> *Delete attachment* in the case comment. 
 
 [role="screenshot"]
-image::images/remove-indicator.png[Removing an indicator from a case]
\ No newline at end of file
+image::images/remove-indicator.png[Removing an indicator from a case]
+
+[discrete]
+[[add-indicator-to-blocklists]]
+== Add indicators to blocklists
+
+Add indicators to blocklists to prevent selected applications from running on your hosts. When creating a blocklist, you can add indicators that have hash values for the MD5, SHA-1, or SHA-256 fields and are of the `file` indicator type.
```

Comment:
```suggestion
Add indicators to blocklists to prevent selected applications from running on your hosts. You can add indicators of the `file` indicator type and that have hash values for the MD5, SHA-1, or SHA-256 fields.
```